sequence/sequences/compose/arith.py

```diff
@@ -0,0 +1,74 @@
+from sequence.core.core import Sequence
+
+import numpy as np
+
+
+class Arith(Sequence):
+    """Sequences combined via arithmetic operation, e.g., addition, multiplication.
+
+    Parameters
+    ----------
+    sequences : List[Sequence]
+        List of sequences to combine.
+    operation : None, str, function, or numpy ufunc, optional, default = None = mean
+        if str, must be one of "mean", "+" (add), "*" (multiply), "max", "min"
+        if func, must be of signature (1D iterable) -> float
+        operation carried out on the sequences
+    """
+
+    def __init__(self, sequences, operation):
+        super().__init__()
+        self.sequences = sequences
+        self.operation = operation
+        self._op = self._resolve_operation(operation)
+
+    def __add__(self, other):
+        if not isinstance(other, Arith) or self._op != other._op:
```

Comment:
why do we have `self._op != other._op`?

Reply:
The line is about dealing with the situation where you have `Arith([a, b, c], sym) + `Arith([d, e, f], sym2)`.

If `sym == sym2`, then you'd want this to become `Arith([a,b,c,d,e,f], sym)` rather than `Arith(Arith([a,b,c],sym), Arith([d,e,f],sym))`, because all `sym` here are commutative. Well, except the mean, but that's an oversight which we probably need to fix in `sktime` as well.

The "or" is for checking that indeed `sym==sym2`. We could do so by the string, but the same operation has multiple string aliases (`"+"` or `"add"`), so we check `_op`.

Reply:
Oh yeah... good point...

But will the final user be able to access this class? Because if the answer is no, we don't have to have different alias for the same operation, not?

Reply:
I thought of it as user facing, potentially. Even if not, it makes sense to minimize the size of the parse tree.